pipeline_annotate.sh

```diff
@@ -53,6 +53,12 @@ get_unique_phenotypes() {
   done
 }
 
+join_arr() {
```

Comment:
Replace https://github.com/molgenis/vip/blob/master/pipeline_preprocess.sh#L86 and https://github.com/molgenis/vip/blob/master/pipeline_preprocess.sh#L160 with a call to joinArr